tools/packaging/kata-deploy/Dockerfile

```diff
@@ -4,19 +4,10 @@
 
 # Specify alternative base image, e.g. clefos for s390x
 ARG IMAGE
-FROM ${IMAGE:-registry.centos.org/centos}:7
-ARG KATA_ARTIFACTS=./kata-static.tar.xz
+FROM ${IMAGE:-docker.io/centos}:7
+ARG KATA_ARTIFACTS=./local-build//kata-static.tar.xz
```

Comment:
Could you explain why we need this?

Reply:
@fidencio  I test it. registry.centos.org/centos is not a manifest tag, i.e. it only provides x86_64 under that tag. docker.io provide all archs, hence go with docker.io

Reply:
Make it east to for developer. With this change, developer just run docker build after all staic files built, no need an extra step to move the static file into target place. Less work, less problem.